std/python/lib/Inspect.hx

```diff
@@ -26,11 +26,11 @@ import python.Tuple.Tuple2;
 
 @:pythonImport("inspect")
 extern class Inspect {
-	static function getmembers(object:Dynamic, ?predicate:Dynamic->Bool):Array<Tuple2<String, Dynamic>>;
-	static function ismethod(object:Dynamic):Bool;
-	static function isclass(object:Dynamic):Bool;
-	static function isfunction(object:Dynamic):Bool;
-	static function getsourcefile(object:Dynamic):String;
+	@:pure(true) static function getmembers(object:Dynamic, ?predicate:Dynamic->Bool):Array<Tuple2<String, Dynamic>>;
+	@:pure(true) static function ismethod(object:Dynamic):Bool;
+	@:pure(true) static function isclass(object:Dynamic):Bool;
+	@:pure(true) static function isfunction(object:Dynamic):Bool;
+	@:pure(true) static function getsourcefile(object:Dynamic):String;
```

Comment:
> This will fail with a [TypeError](https://docs.python.org/3/library/exceptions.html#TypeError) if the object is a built-in module, class, or function

That last one isn't pure
https://docs.python.org/3/library/inspect.html#inspect.getsourcefile